src/pages/bounties/legendre-prf/concrete-instance-bounties.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/no-children-prop */
```

Comment:
Do we need a standalone React component for each page here? I was hoping for something similar to the `[slug].tsx` setup you implemented for blog posts, where we just have one React component file serve as a template for all bounty pages. Seems redundant/unnecessary to have a React component for each of these.

Reply:
The file structure is different, some bounties have subpages, some don't, the logic would be way more complex.